docs/amko_tenancy.md

```diff
@@ -0,0 +1,38 @@
+# Tenancy support in AMKO
+
+This feature allows AMKO to create GSLB object in a user-specific tenant in Avi.The expected isolation and administrative restrictions of a multi-tenant architecture in NSX Advanced Load Balancer GSLB extend to AMKO
+
+## Tenant Context
+
+AVI non admin tenants primarily operate in 2 modes, **provider context** and **tenant context**.
+
+### Provider Context
```

Comment:
I dont think this needs to be mentioned here, I dont see an impact on AMKIO for this

Reply:
removed this section